### ISSUE TITLE
Override maximum-point-size and minimum-point-size per style?

### ISSUE THREAD

**Issue by avi-screenovate** [rwbutler/TypographyKit#7]
Is it possible to override maximum-point-size and minimum-point-size per style?

**Comment by rwbutler** [comment]
There isn't yet but this is something I was planning to implement - I'll take a look at adding this feature soon.

**Comment by rwbutler** [comment]
Just to let you know I've now implemented this feature in 3.3.0. You should be able to add this using the following JSON:

```json
		"paragraph": {
			"font-name": "Avenir-Medium",
            "minimum-point-size": 12,
            "maximum-point-size": 24,
			"point-size": 18,
			"text-color": "text",
			"letter-case": "regular"
		}
```
